src/main/java/globalquake/ui/database/StationSourcesPanel.java

```diff
@@ -19,6 +19,8 @@ public class StationSourcesPanel extends JPanel {
     private final UpdateStationSourceAction updateStationSourceAction;
     private final JTable table;
     private StationSourcesTableModel tableModel;
+    private JButton btnSelectStations;
+    private JButton btnLaunch;
```

Comment:
These fields can be converted to local variables (btnSelectStations and btnLaunch) or even better - addressed directly by your getters